lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -235,18 +235,18 @@ void StagesSubject::stageChanged(
             // We need to send some notifs so Maya can update (such as on undo
```

Comment:
Just above here, I could modify the loop over GetResyncedPaths to get access to the SdfChangeList::entry objects and access the same flags I get in the changedInfoOnlyPaths loop.